src/sentry/web/frontend/release_webhook.py

```diff
@@ -77,11 +77,15 @@ def _handle_builtin(self, request, project):
     def post(self, request, plugin_id, project_id, signature):
         project = Project.objects.get_from_cache(id=project_id)
 
-        token = ProjectOption.objects.get_value(project, 'sentry:release-token')
-
         logger.info('Incoming webhook for project_id=%s, plugin_id=%s',
                     project_id, plugin_id)
 
+        token = ProjectOption.objects.get_value(project, 'sentry:release-token')
+
+        if token is None:
+            logger.warn('No token for release hook')
```

Comment:
tbh I’d argue this logging here is pretty useless. It provides no information to help debug. It doesn’t have a project id or any identifying information.